app/assets/stylesheets/scaffolds.scss

```diff
@@ -86,10 +86,6 @@ body.dark-theme {
 }
 
 .trusted-visible-block {
-  display: none !important;
-}
-
-body.trusted-status-true .trusted-visible-block {
```

Comment:
These lines should remain in the CSS as per the suggestion mentioned above

Reply:
Reverted this change